### PR TITLE
feature/AT-RSC-UI-Added Mentors View

### PR DESCRIPTION
**Pull request by Normaillescas29**
Added Mentors View

### CODE REVIEW

**Review by irvingesp2190 — APPROVED** [review]

File: src/routes/at-resources.routes.js [hunk 1]
```diff
@@ -13,7 +13,8 @@ const { dashboard,
         renderAddSprintForm,
         addSprint,
         renderEditSprintForm,
-        updateSprint} = require("../controllers/at-resources.controller");
+        updateSprint,
+        renderMentorList,} = require("../controllers/at-resources.controller");
```
Comment (at-internship) [inline src/routes/at-resources.routes.js]: should be `renderMentorList} = require("../controllers/at-resources.controller");`

File: src/views/mentors.hbs [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<div class="container-fluid">
```
Comment (at-internship) [inline src/views/mentors.hbs:1]: are you sure this works?

**Review by at-internship — CHANGES_REQUESTED** [review]

File: src/routes/at-resources.routes.js (hunk 1 above)
Comment (Normaillescas29) [reply src/routes/at-resources.routes.js]: Correction ready

File: src/views/mentors.hbs (hunk 2 above)
Comment (Normaillescas29) [reply src/views/mentors.hbs:1]: Correction ready

**Comment by irvingesp2190** [comment]
Corrections ready, @at-internship  Can you please help us with your approval to proceed with merging? Thanks

File: src/views/mentors.hbs [hunk 3]
```diff
@@ -1,5 +1,4 @@
-<div class="container-fluid">
-    <div class="row">
+    <div class="row"></div>
```
Comment (at-internship) [inline src/views/mentors.hbs]: This hbs template is incompete

**Review by at-internship — CHANGES_REQUESTED** [review]

File: src/views/mentors.hbs (hunk 3 above)
Comment (Normaillescas29) [reply src/views/mentors.hbs]: Template modded

**Review by at-internship — APPROVED** [review]
Approved

**Pull request merged**